index.js

```diff
@@ -124,11 +125,13 @@ class UpdateNotifier {
 
 		options = {
 			isGlobal: isInstalledGlobally(),
+			isYarnGlobal: isYarnGlobal()(),
 			...options
 		};
+		const installCommand = options.isYarnGlobal ? `yarn global add ${this.packageName}` : `npm i ${options.isGlobal ? '-g ' : ''}${this.packageName}`;
```

Comment:
Shouldn't we also show local Yarn command?

Reply:
> Shouldn't we also show local Yarn command?
@sindresorhus How do we know if it is using Yarn or NPM locally?

Reply:
https://github.com/sindresorhus/has-yarn